hosting/templates/hosting/payment.html

```diff
@@ -49,6 +49,15 @@
                 </div>
             </div>
         </div>
+        <div class="help-block with-errors">
```

Comment:
IMO, the messages would look better at the top before the `payment-container`. @verysanghee what do you think?